mgmtsystem_nonconformity/readme/USAGE.md

```diff
@@ -0,0 +1,48 @@
+To use this module:
+
+- Go to Management System \> Nonconformities
+- Click on Create to enter the following information:
+- Partner : Customer, supplier or internal personnel
+- Related to: Any reference pointing to the NC (order id, project id,
+  task id, etc.)
+- Responsible: Person responsible for the NC
+- Manager : Person managing the department or owner of the procedure
+- Filled in by: Originator of NC report
+- Origins: The source of the NC, how was it discover
+- Procedures: Against which procedure is the NC
+- Description: Evidence, reference to the standards
+- Click on Save and then on Analysis.
```

Comment:
```suggestion
- Click on Save and then set state from Draft to Analysis.
```

minor suggestion, as on runbot tab Causes and Analysis appears when state is changed.